.travis.yml

```diff
@@ -5,19 +5,12 @@ env:
 language: ruby
 script: "bundle exec rake ci"
 rvm:
-  - 1.8.7
-  - 1.9.3
-  - 2.0.0
-  - jruby-19mode
-  - jruby-head
-  - rbx-19mode
+  - '2.1.6'
```

Comment:
My mistake. In https://github.com/backus/yardstick/commit/1dc4c6c928089fd80d166172d9e35a7f95af7f05 it's explicitly stated that 2.1.6 was used due to an error in Parser.